samples/bluetooth/broadcast_audio_sink/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -1,7 +1,5 @@
 # The LC3 codec uses a large amount of stack. This app runs the codec in the work-queue, hence
-# inctease stack size for that thread.
+# increase stack size for that thread.
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
```

Comment:
Hmm, the comment here is actually no longer true. The LC3 decoder runs in its own thread in the sample now. Perhaps the increase to `CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` should just be removed

Reply:
Removed... the sample seems to build and work without this increase.